src/deprecated.jl

```diff
@@ -32,6 +32,9 @@
 @deprecate setstore(T::ITensor, st) setstorage(T, st) false
 @deprecate uniqueindex(args...; kwargs...) uniqueind(args...; kwargs...)
 
+# itensor/emptyITensor
+@deprecate emptyITensor(args...; kwargs...) ITensor(args...; kwargs...)
```

Comment:
Note that with this, we should be able to just delete all of the code for `emptyITensor` elsewhere, presuming that its functionality is subsumed by the `ITensor` constructor.

Reply:
Makes sense, thanks!